main.bundle.js

```diff
@@ -186,8 +189,37 @@
 	    });
 	  }
 
+	  createMealButtonListener() {
+	    let meals = ["Breakfast", "Lunch", "Dinner", "Snack"];
+	    meals.forEach(meal => {
+	      $(`.new${meal}`).on('click', function () {
+	        $('.displayFoods').show();
+	        $('.addToMeal').show();
+	        $(`.new${meal}`).css({ 'background-color': "yellow" });
```

Comment:
The spec says to have the breakfast, etc. buttons add the foods to that specific meal.